core/kernel/initcall.c

```diff
@@ -4,10 +4,18 @@
  * Copyright (c) 2014, STMicroelectronics International N.V.
  */
 
+#if defined(ARM32) || defined(ARM64)
+#include <arm.h>
+#endif
 #include <initcall.h>
 #include <trace.h>
 #include <kernel/linker.h>
 
+/* Define this manually on non-arm architectures */
+#ifndef __pauth
+#define __pauth
+#endif
```

Comment:
I wonder if `__pauth` macro should rather be defined in _lib/libutils/ext/include/compiler.h_. @jforissier, what do you think?

Reply:
Renamed to __kernel_nopauth  according Jerome's advice.